Main1.cpp

```diff
@@ -1,31 +1,35 @@
 #ifndef Main1_cpp
 #define Main1_cpp
-#include<iostream>
-#include "Draw.cpp"
+#include <iostream>
+#include "Shape.cpp"
 #include "Enum.cpp"
 #include "Square.cpp"
 #include "Rectangle.cpp"
 #include "Circle.cpp"
 #include "ShapeFactory.cpp"
+#include "Area.cpp"
 
 using namespace std;
 
 int main()
 {
     string shape;
-    
+
     cout << "Enter a Shape" << endl;
     cin >> shape;
 
-    Draw* shapeDrawer = ShapeFactory :: selectShape(shape);
+    Shape *shapeDrawer = ShapeFactory ::selectShape(shape);
+
+    int area = shapeDrawer->CalculateArea();
```

Comment:
Result from function invocation `CalculateArea` is never used. Do we need to use the result? If yes, then add the statements which are using `area`, If no, then remove the area variable from here.